src/Microsoft.AspNetCore.All/build/PublishWithAspNetCoreTargetManifest.targets

```diff
@@ -0,0 +1,49 @@
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <PropertyGroup>
+    <PublishWithAspNetCoreTargetManifest Condition="'$(PublishWithAspNetCoreTargetManifest)'==''">true</PublishWithAspNetCoreTargetManifest>
+  </PropertyGroup>
+
+<!--
+*****************************************************************************
+Target: PublishWithAspNetCoreTargetManifest
+Append the default ASP.NET Core runtime package store manifest during publish
+*****************************************************************************
+-->
+  <Target
+    Name="PublishWithAspNetCoreTargetManifest"
+    BeforeTargets="Publish"
+    DependsOnTargets="PrepareForPublish"
+    Condition="'$(PublishWithAspNetCoreTargetManifest)'=='true' and '$(PublishableProject)'=='true'" >
+
+    <PropertyGroup>
+      <ManifestRuntimeIdentifier>$(AspNetCoreTargetManifestRuntimeIdentifier)</ManifestRuntimeIdentifier>
+      <ManifestRuntimeIdentifier Condition="'$(AspNetCoreTargetManifestRuntimeIdentifier)'==''
```

Comment:
Can we declare this in a separate propertygroup so we shouldn't have to repeat `'$(AspNetCoreTargetManifestRuntimeIdentifier)'==''`?